appveyor.yml

```diff
@@ -28,8 +32,28 @@ before_build:
   - flake8 ./scripts/
 
 build_script:
-  - cmake .  -DCMAKE_BUILD_TYPE=%CONFIG% -G "%GENERATOR%"
+  # Request `libcmt.lib` is used so our released artifacts don't dynamically
+  # link to `msvcrt.dll`
+  - cmake .  -DCMAKE_BUILD_TYPE=%CONFIG% -G "%GENERATOR%" -DMSVC_USE_LIBCMT=YES
   - cmake --build . --config %CONFIG% -- %PARALLEL_FLAG%%DEGREE_OF_PARALLELISM%
 
 test_script:
   - ctest --output-on-failure --timeout 10 -j 5
+
+before_deploy:
+  - ps: |
+        $NAME = "binaryen-${env:APPVEYOR_REPO_TAG_NAME}-${env:ARCH}-windows"
+        Move-Item -Path bin -Destination $NAME
+        7z a -ttar "${NAME}.tar" "${NAME}"
+        7z a "${NAME}.tar.gz" "${NAME}.tar"
+        Push-AppveyorArtifact "${NAME}.tar.gz"
+
+deploy:
+  artifact: /.*\.tar.gz/
+  auth_token:
+    secure: dtHSvbZkdAFtL0J5YrSw8DpxjfYuHWgqD1SupmJT/yfYSjEBiX55RFXRoqBM2tx1
```

Comment:
This'll need to be filled in with an actual value before merging!

Reply:
ok the encrypted value should be `zM0Bcjy1JXOBuu2C32lY0vCxREu7ah+bYFUpwmuryw82+HgCjvq7ZMutAk34Lv9d`

Reply:
Thanks!